libs/build.js

```diff
@@ -0,0 +1,133 @@
+const path = require('path');
+const utils = require('./utils');
+const template = require('./template');
+const memo = require('./memo').memo;
+
+const DEFAULT_IGNITOR_EVENT = {
+  "ignitor": true,
+}
+
+const DEFAULT_FUNCTION_OPTIONS = {
+  name: '.*',
+  schedule: true,
+}
+
+const DEFAULT_IGNITOR_SCHEDULE = {
+  rate: 'rate(5 minutes)',
+  enabled: true,
+  input: DEFAULT_IGNITOR_EVENT
+};
+
+const DEFAULT_OPTIONS = {
+  functions: [
+    DEFAULT_FUNCTION_OPTIONS,
+  ],
+};
+
+// !!WARNING!! Do not include a '.' in the directory name 
```

Comment:
Did you ever figure out why this didn't work?

Reply:
Yes, it looks like packaging is fine, although I was getting similar errors when originally making this. However commands like `sls invoke local` will fail because the way serverless finds and invokes the function is based on the same way I create templates form filename, and module. 

example
```handler: src/handlers.default```

This is turned into:
**filename** = _src/handlers_
**module** = _default_

The issue is that serverless sees the overridden handler `.ignitor/wrapped.default` and splits on the first period. 

It then resolves the files into:
 **filename** = _''_
**module** = _ignitor/wrapped.default_

... and subsequently throws **MODULE_NOT_FOUND**, I believe the same thing happens in the deployed code where it cannot find the module as well.